pkgs/applications/audio/snapcast/default.nix

```diff
@@ -9,24 +9,15 @@ assert pulseaudioSupport -> libpulseaudio != null;
 
```

Comment:
The assert can also be dropped. libpulseaudio is never null unless it is explicitly overwriten to that and then people probably know what they are doing.

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/305661/commits/5c4361ae2fc982934e13dd30983fdc7764e0a1ca